game/gamestates/start_menu.lua

```diff
@@ -0,0 +1,92 @@
+--MODULE FOR THE GAMESTATE: GAME--
+local MENU = require 'infra.menu'
+local INPUT = require 'infra.input'
+local CONTROLS = require 'infra.control'
+local HudView = require 'domain.view.hudview'
+
+local state = {}
+
+--LOCAL VARIABLES--
+
+local _width, _height
+local _menu_view
+local _font
+
+--LOCAL FUNCTIONS--
+
+local function _sendToLayer ()
+  local render_queue = MENU.getRenderQueue()
+  while not render_queue.isEmpty() do
+    local printing = render_queue.pop()
+    _menu_view:push(printing)
+  end
+end
```

Comment:
The could be kept in MENU with a `flush()` function that receives the `menu_view`